src/main/kotlin/no/digdir/service_catalog/model/PublicService.kt

```diff
@@ -16,3 +16,9 @@ data class PublicService (
     val title: LocalizedStrings?,
     val description: LocalizedStrings?
 )
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class PublicServiceToBeCreated(
+    val title: LocalizedStrings,
```

Comment:
Du kan like så greit bare holde denne nullable. Alt internt i LocalizedStrings er nullable, så det vil uansett være mulig å opprette en uten tittel.
```suggestion
    val title: LocalizedStrings?,
```